externs/olx.js

```diff
@@ -4975,14 +4974,6 @@ olx.source.ImageStaticOptions.prototype.imageExtent;
 
 
 /**
- * Size of the image in pixels.
- * @type {ol.Size|undefined}
- * @api stable
- */
-olx.source.ImageStaticOptions.prototype.imageSize;
```

Comment:
Looks like it makes sense then to bring the imageSize property back. Will incorporate all of @tschaub's suggestions in the follow up pull request.